en/asset/prefab.md

```diff
@@ -1,47 +1,63 @@
+## Status of prefab nodes
 
-__Third__, locating assets is convenient to quickly locate __Prefab assets__ in the __Asset Manager__. When there are many assets, the efficiency is higher.
+__Prefab Nodes__ in the __Inspector__ panel render __green__ to indicate normal association with assets and render __red__ to indicate that the associated assets no longer exist.
 
-__Fourth__, you can deserialize a __Node__ object again from the asset and replace the existing node. This operation does not change the *uuid* of the existing __Node__. It is usually used when you want the __Node__ to be re-synchronized with the __Prefab Asset__.
 
-__Fifth__, when there are new changes to the __Node__, and you want to update to the associated __Prefab__ asset, use this button.
+## Editing Prefab Nodes in a Scene
 
-## Status of prefab nodes
+### General Operations
+In the __Hierarchy__ panel, select the __Prefab Node__, and notice there are several buttons that can be clicked at the top of the __Inspector__ panel. If you move the mouse over, there will be a text prompt for these functions. The following image describes the functions of the buttons in the red frame area:
 
-__Prefab Nodes__ in the __Inspector__ panel render __green__ to indicate normal association with assets and render __red__ to indicate that the associated assets no longer exist.
+![](prefab/edit-prefab.png)
 
-## Entering prefab editing mode
+Icon|Description
+---|---
+![](prefab/unlink-prefab-button.png)|Revert to __normal node__. __Prefab Instance__ can become ordinary __Nodes__, that is, completely separated from the relationship between assets. This function is also available in the top-level menu `Node`.
+![](prefab/locate-prefab-button.png)|Locating assets.Its convenient to quickly locate __Prefab Asset__ in the __Assets__ panel.
+![](prefab/revert-prefab-button.png)|Restore from asset. Restore the data of the current __Prefab Instance__ to the data in the __Prefab Asset__, but the name, location, and rotation will not be restored.
+![](prefab/apply-prefab-button.png)|Update to asset. Update all data of the current __Prefab Instance__ to the associated __Prefab Asset__.
 
-__Double-click__ the __Prefab Asset__ in the __Assets__ panel to switch from __Scene__ editing mode to __Prefab editing__ mode. You can edit the __Prefab Assets__ in the __Editor__. After editing, click __Save Prefab__ in the __Scene editor__ to save the edited __Prefab Assets__. __Next__, click __Close__ to return to the __Scene__ editing mode.
+### Add New Node
+A new node added under the __Prefab Instance__ will have a `+` sign in the lower right corner of the node name. Its data is stored under the __Prefab Instance__, so it will not affect the data of the associated __Prefab Asset__.

```

Comment:
```suggestion

A new node added under the __Prefab Instance__ will have a **+** sign in the lower right corner of the node name. Its data is stored under the __Prefab Instance__, so it will not affect the data of the associated __Prefab Asset__.
```